### PR TITLE
Android 12 - 12-3.0 - new backup format!

### PR DESCRIPTION
**Pull request by chirayudesai**
* Bump version following versioning scheme
* Add initial entry to changelog

Change-Id: Ifca1b3feac0f510890450a1999b42897894d58ee

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## [12-2.2] - 2021-09-29
+* Initial release for Android 12
```
Comment (grote) [inline CHANGELOG.md:2]: The initial release for Android 12 will hopefully be `12-3.0`

Comment (chirayudesai) [reply CHANGELOG.md:2]: Alright let's just leave this one open for now then.

**Comment by chirayudesai** [comment]
Need to add more to the changelog, can either do it now, or before we tag the stable release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## [12-3.0] - 2021-09-29
+* Initial release for Android 12
+* New backup format, identical to the storage backup
```
Comment (grote) [inline CHANGELOG.md]: Well, not really identical, but it uses the same crypto. Better hides what apps get backed app (still visible to storage medium though), overall speed-up of backup.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+## [12-3.0] - 2021-09-29
```
Comment (grote) [inline CHANGELOG.md]: maybe push the date a bit to the future instead of the past?

File: CHANGELOG.md (hunk 2 above)
Comment (chirayudesai) [reply CHANGELOG.md]: Updated based on what you wrote here and in the PR.

File: CHANGELOG.md (hunk 3 above)
Comment (chirayudesai) [reply CHANGELOG.md]: This was set to the date I made the commit. I set it to today for now, can always bump on day of tag.

**Comment by chirayudesai** [comment]
Added some more bits.

I'll re-edit the Changelog before tagging, at that point I can go through the git log and add anything that's missed.

**Review by grote — APPROVED** [review]

**Pull request merged**